cmd/cmd.go

```diff
@@ -129,3 +108,47 @@ func help(cmd *cobra.Command, _ []string) {
 		fmt.Println("Error while printing help: ", err.Error())
 	}
 }
+
+// Checks if the version check should be run
+func shouldRunVersionCheck(skipVersionCheckFlag bool, commandName string) bool {
+
+	// If either are true, then the version check should NOT run, hence negation
+	return !(skipVersionCheckFlag || canCommandSkipVersionCheck(commandName))
```

Comment:
This benefits from the fact that `!A AND !B` is equivalent to `!(A or B)` and the latter benefits from Short circuit evaluation, as shown by this small test https://go.dev/play/p/M0gfXelQf2y